src/components/TimeSelectInput/helpers/index.js

```diff
@@ -0,0 +1,39 @@
+import get12HourTime from './get12HourTime';
```

Comment:
here you can:
```
export { default as get12HourTime } from './get12HourTime';
export { default as get24HourTime } from './get24HourTime';
...
```
and then it gets shorter importing and exporting in only one line each one